src/pages/Review/Review.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import Total from './Total';
+import { withRouter } from 'react-router-dom';
+import ReviewList from './ReviewList';
+import { BASE_URL } from '../../config.js';
+import './Review.scss';
+
+class Review extends Component {
+  constructor() {
+    super();
+    this.state = {
+      reviewList: [],
+      itemId: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch(`${BASE_URL}/products/${this.props.match.params.id}`)
+      .then(data => data.json())
+      .then(data => {
+        console.log(data.RESULT);
+        this.setState({
+          reviewList: data.RESULT.review,
+          itemId: this.props.match.params.id,
+        });
+      });
+  }
+
+  reviewWriteBtn = () => {
+    fetch(`${BASE_URL}/products/${this.props.match.params.id}`, {
+      method: 'POST',
+      body: JSON.stringify({
+        //token: localStorage.getItem('TOKKEN'),
+        itemId: this.state.itemId,
+      }),
+      headers: {
+        Authorization: localStorage.getItem('TOKKEN'),
+        Accept: 'application / json',
+      },
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.MESSAGE === 'UNATHORIZED') {
+          alert('리뷰 권한이 없습니다!');
+          return;
+        }
+        if (result.MESSAGE === 'INVALIDTOKEN') {
```

Comment:
```suggestion
        if (result.MESSAGE === 'UNAUTHORIZED') {
          alert('리뷰 권한이 없습니다!');
          return;
        }
        if (result.MESSAGE === 'INVALID_TOKEN') {
```
- 오타 수정하시고 띄어 쓰기 필요한 곳에서는 언더스코어 잘 넣어주세요

Reply:
이것도 백엔드 분들에게 말씀드려 수정완료했습니다!